helmfile/charts/grafana-ops/values.yaml

```diff
@@ -26,3 +26,32 @@ dashboards:
     enabled: true
   networkpolicy:
     enabled: true
+  daily:
+    enabled: true
+
+daily:
+  limits:
+    influxdb:
+      workloadClusterKB: 4500000
+      serviceClusterKB: 4500000
+  thresholds:
+    # Thresholds for the number of elasticsearch snapshots
+    elasticsearchSnapshots:
+      critical: 20
+      warning: 16
+    # How old should the oldest snapshot get before it indicates an issue?
+    # Daily backups + retention of max 14 snapshots means maximum 15 days.
+    # With time based retention set to 10 days, it should be no more than 11.
+    elasticsearchOldestSnapshotSeconds:
```

Comment:
It depends, we might have a bug where a snapshot is taken every 10'th minute meaning that snapshots will most likely be removed due to `nr of snapshots > max snapshots` so no snapshot will ever be "old".
But then again the converse can also happen - too few snapshots are taken/failing, meaning that no snapshot will be removed due to `nr of snapshots < min snapshots`.

I guess in essence I don't know there answer to your question.

Reply:
Yeah this is complicated. I was first thinking that it would be great to calculate this number based on the values we already have. But I think that would be way too hard, and basically require us to parse the cron-schedule. So this is probably the best we can do. Let the user set the time. :slightly_smiling_face:

Reply:
Agree